src/Apps/Artwork/Components/TrustSignals/VerifiedSeller.tsx

```diff
@@ -0,0 +1,41 @@
+import { VerifiedIcon } from "@artsy/palette"
+import { VerifiedSeller_artwork } from "__generated__/VerifiedSeller_artwork.graphql"
+import React from "react"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "react-relay"
+import { TrustSignal, TrustSignalProps } from "./TrustSignal"
+
+interface VerifiedSellerProps
+  extends Omit<TrustSignalProps, "Icon" | "label" | "description"> {
+  artwork: VerifiedSeller_artwork
+}
+
+export const VerifiedSeller: React.FC<VerifiedSellerProps> = ({
+  artwork,
+  ...other
+}) => {
+  return (
+    artwork.partner.isVerifiedSeller && (
```

Comment:
We should probably be safe here `artwork.partner && artwork.partner.isVerifiedSeller`

Reply:
```suggestion
    artwork.partner && artwork.partner.isVerifiedSeller && (
```